setup.cfg

```diff
@@ -66,6 +67,7 @@ omit =
   utils/utils.py
   plugins/ban.py
   plugins/labhub.py
+  plugins/wolfram_alpha.py
```

Comment:
Try to fix the issue instead of ignoring it if possible. We will resort to ignoring it as a last option.

Reply:
Since, I have ignored the wolfram_alpha_test, it is giving no coverage for this plugin, so just ignored it. Once the wolfram_alpha_test is fixed, both the files will be added to the tests.

Reply:
The fix should go before we merge this then.

Reply:
Sure, will do this.

Reply:
I will get this one done and then squash all the commits.

Reply:
Done with the changes. The builds are working and no file is ignored. Also, no upgrade command in travis.yml in script section.